### PR TITLE
New recipe xlnt/1.5.0

### PR DESCRIPTION
**Pull request by SpaceIm**
Specify library name and version:  **lib/1.0**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by uilianries — DISMISSED** [review]

**Comment by SpaceIm** [comment]
it will be possible to unvendor libstudxml after https://github.com/conan-io/conan-center-index/pull/14185

**Comment by SpaceIm** [comment]
waiting https://github.com/conan-io/conan-center-index/pull/14185

File: recipes/xlnt/all/conandata.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+sources:
+  "1.5.0":
+    url: "https://github.com/tfussell/xlnt/archive/refs/tags/v1.5.0.tar.gz"
+    sha256: "8dec266d59ab6e4829da5dacea764a02887eeff5a5501c9a51ce796e735b90de"
+patches:
+  "1.5.0":
+    - patch_file: "patches/0001-fix-cmake-and-unvendor-deps.patch"
+      patch_description: "Fix CMakeLists and unvendor miniz & utfcpp"
```
Comment (SpaceIm) [inline recipes/xlnt/all/conandata.yml:8]:
```suggestion
      patch_description: "Fix CMakeLists and unvendor dependencies"
```

File: recipes/xlnt/all/conandata.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    url: "https://github.com/tfussell/xlnt/archive/refs/tags/v1.5.0.tar.gz"
+    sha256: "8dec266d59ab6e4829da5dacea764a02887eeff5a5501c9a51ce796e735b90de"
+patches:
+  "1.5.0":
+    - patch_file: "patches/0001-fix-cmake-and-unvendor-deps.patch"
+      patch_description: "Fix CMakeLists and unvendor dependencies"
+      patch_type: "conan"
+    - patch_file: "patches/0002-include-limits.patch"
+      patch_description: "Add missing includes"
+      patch_type: "conan"
```
Comment (prince-chrismc) [inline recipes/xlnt/all/conandata.yml:12]:
```suggestion
      patch_type: "portability"
```

File: recipes/xlnt/all/conanfile.py [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+        self.requires("libstudxml/1.1.0-b.10+1")
+        self.requires("miniz/3.0.1")
+        self.requires("utfcpp/3.2.2")
+
+    def validate(self):
+        if self.info.settings.compiler.get_safe("cppstd"):
+            check_min_cppstd(self, 11)
+        libstudxml_version = Version(self.dependencies["libstudxml"].ref.version)
+        libstudxml_major_minor = f"{libstudxml_version.major}.{libstudxml_version.minor}"
+        if Version(libstudxml_major_minor) < "1.1":
```
Comment (prince-chrismc) [inline recipes/xlnt/all/conanfile.py:54]: I was uned the impression the new version helper should ignore the prerelease and build information 🤔 
Did you try without this or just went to try and trusted 1.x impl?

Curious if there's a bug :wink

Comment (uilianries) [reply recipes/xlnt/all/conanfile.py:54]: Thank you for checking!

**Review by uilianries — APPROVED** [review]
LGTM

**Review by prince-chrismc — APPROVED** [review]

**Comment by SpaceIm** [comment]
Is CI stuck?

**Comment by uilianries** [comment]
I just restarted it now. Let's see.

**Comment by prince-chrismc** [comment]
There's 500 + things in the queue right now :) be gentle please

**Comment by SpaceIm** [comment]
> There's 500 + things in the queue right now :) be gentle please

Seems to be unrelated. New PRs build fine and quickly, while old PRs which should have precedence struggle.

**Comment by uilianries** [comment]
I just cancelled old CI builds which were running since Sunday (llvm-core for instance). Let's see if this unblocks the working queue now.

**Comment by SpaceIm** [comment]
Still stuck :(

**Comment by prince-chrismc** [comment]
My turn to kick a bunch of PRs

image

**Pull request merged**